src/project/internal/notationproject.cpp

```diff
@@ -37,6 +37,7 @@
 #include "notation/notationerrors.h"
 #include "projectaudiosettings.h"
 #include "projectfileinfoprovider.h"
+#include "project/iprojectautosaver.h"
```

Comment:
You should not need `project/` here, because `project` is an include directory for the target of which this file is part.

Reply:
Removed the prefix.